src/mlpack/core/dists/gamma_distribution.hpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file gamma_distribution.hpp
+ * @author Yannis Mentekidis
+ *
+ * Implementation of a Gamma distribution of multidimensional data that fits
+ * gamma parameters (alpha, beta) to data. Assumes each data dimension is
+ * uncorrelated.
+ *
+ * Based on "Estimating a Gamma Distribution" by Thomas P. Minka:
+ * research.microsoft.com/~minka/papers/minka-gamma.pdf
+ */
+
+#ifndef _MLPACK_CORE_DISTRIBUTIONS_GAMMA_DISTRIBUTION_HPP 
+#define _MLPACK_CORE_DISTRIBUTIONS_GAMMA_DISTRIBUTION_HPP
+
+#include <mlpack/core.hpp>
+namespace mlpack{
+namespace distribution{
+
+/**
+ * Class for fitting the Gamma Distribution to a dataset.
```

Comment:
We should point out here that this is a multidimensional Gamma distribution, but it assumes that each dimension is fully independent.